pkg/proxy/namespace/namespace.go

```diff
@@ -1,11 +1,53 @@
 package namespace
 
-import "net/url"
+import (
+	"context"
+	"fmt"
+	"net/url"
+
+	"github.com/codeready-toolchain/registration-service/pkg/log"
+
+	authenticationv1 "k8s.io/api/authentication/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
 
 // NamespaceAccess holds an information needed to access the namespace in a member cluster for the specific user
 type NamespaceAccess struct { // nolint:revive
 	// APIURL is the Cluster API Endpoint for the namespace
-	APIURL url.URL
+	apiURL url.URL
```

Comment:
what's the benefit of using private field along with a getter? do you need to protect the write-access to these fields?

Reply:
Yes. Wanted to protect the write-access.

Reply:
ok